apollo-portal/src/main/resources/static/views/component/release-modal.html

```diff
@@ -216,9 +220,14 @@ <h4 class="modal-title" ng-show="toReleaseNamespace.isBranch && toReleaseNamespa
                         </table>
 
                         <!--file format -->
+                        <div ng-repeat="config in toReleaseNamespace.branchItems" ng-if="!toReleaseNamespace.isPropertiesFormat&&toReleaseNamespace.isBranch"
+                             ng-show="releaseChangeViewType=='CompareWithMasterValue'">
+                            <apollodiff old-str="config.masterReleaseValue" new-str="config.newValue" apollo-id="'releaseStrDiff1'">
```

Comment:
It seems that 'apollo-id' is not being used. Can it be removed? Also, the naming of 'releaseStrDiff1' and 'releaseStrDiff2' is not very appropriate.